src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -761,29 +778,33 @@ private void RefreshPortColors()
                 // Special case for keeping list structure visual appearance
                 if (_port.UseLevels && _port.KeepListStructure && _port.IsConnected)
                 {
-                    PortBackgroundColor = new SolidColorBrush(Color.FromRgb(94, 165, 196));
```

Comment:
Can these colors be stored as a resource or at least defined statically in this view model.  This was @mjkkirschner observation that this might be a memory leak issue as we keep re-allocating the colors

Reply:
This is an issue we encountered before - when running a many NUNit tests, none of the SharedDictionaryManager dictionaries (containing these colours) would load in. Attempting to reference them would cause an exception. They've been hard-coded for now to unblock progress but it's not a clean solution.

Reply:
you can recycle the colors objects - avoid allocating them over and over again.